docs/kratos/self-service/flows/account-recovery-password-reset.mdx

```diff
@@ -0,0 +1,237 @@
+
+## Account recovery addresses
+
+To provide the account recovery mechanism, Ory Identities (Ory Kratos) needs to know which address to send the recovery message
+to. In most cases, this is the address that the user used when registering their account. But other fields inside the `traits`
+section are supported as well.
+
+:::note
+
+If the email address used for recovery purposes is the same as the email used for verification purposes and the account hasn't
+been activated yet (see [Account Activation](./verify-email-account-activation.mdx)), the address will also be verified by
+completing the account recovery successfully.
+
+:::
+
+To specify that a trait of the identity should be used for recovery, use the following identity schema:
+
+```diff
+ {
+   "$id": "https://schemas.ory.sh/presets/kratos/quickstart/email-password/identity.schema.json",
+   "$schema": "http://json-schema.org/draft-07/schema#",
+   "title": "Person",
+   "type": "object",
+   "properties": {
+     "traits": {
+       "type": "object",
+       "properties": {
+         "email": {
+           "type": "string",
+           "format": "email",
+           "ory.sh/kratos": {
+             "credentials": {
+               "password": {
+                 "identifier": true
+               }
+             },
++            "recovery": {
++              "via": "email"
++            }
+           }
+         }
+       }
+       "additionalProperties": false
+     }
+   }
+ }
+```
+
+## Email templates
+
+<p>
+  <figure>
+    <img alt="Recovery email sent to unknown address" src={useBaseUrl("images/kratos/email-recovery-unknown.png")} />
+    <figcaption>
+      If the requested email address is an unknown recovery address, an account recovery attempt email is sent to that email
+      address.
+    </figcaption>
+  </figure>
+</p>
+
+This prevents account enumeration attacks as explained in this
+[blog post by Troy Hunt](https://www.troyhunt.com/website-enumeration-insanity-how-our-personal-data-is-leaked/).
+
+<p>
+  <figure>
+    <img alt="Recovery email sent to a known recovery address" src={useBaseUrl("images/kratos/email-recovery-known.png")} />
+    <figcaption>
+      If the requested email address is a known recovery address, a recovery code is sent to that email address.
+    </figcaption>
+  </figure>
+</p>
+
+The emails are using templates that can be customized as explained in
+[Customizing E-Mail Templates](../../emails-sms/custom-email-templates).
+
+## Updating credentials
+
+After the user completed the recovery successfully they receive a privileged session and are redirected to the settings page. Here
+they can update their credentials.
+
+Read more about the [privileged sessions](../../session-management/01_overview.mdx#privileged-sessions).
+
+## Invalidate other sessions
+
+In some scenarios it can be useful to deactivate all active sessions, once an account has been recovered. This forces anyone that
+currently has access to the account to re-authenticate.
```

Comment:
```suggestion
In some scenarios it can be useful to revoke all active sessions once an account has been recovered. This forces anyone with access to the account to re-authenticate.
```